google/cloud/security/inventory/pipelines/load_bigquery_datasets_pipeline.py

```diff
@@ -165,8 +170,7 @@ def run(self):
         """Runs the actual data fetching pipeline."""
         dataset_project_access_map = self._retrieve()
 
-        loadable_datasets = self._transform(dataset_project_access_map)
-
-        self._load(self.RESOURCE_NAME, loadable_datasets)
-
-        self._get_loaded_count()
+        if dataset_project_access_map:
```

Comment:
nit: This is the pythonic way to check for None.
https://stackoverflow.com/a/3965129/2830207